CHANGELOG.rst

```diff
@@ -7,6 +7,9 @@ CHANGELOG
 
 * doc-fix: update information about saving models in the MXNet README
 * doc-fix: change ReadTheDocs links from latest to stable
+* feature: Make predictor to delete endpoint configuration by default when calling ``delete_endpoint()``
```

Comment:
Support to for predictor class to delete endpoint configuration by default when calling ``delete_endpoint()``